### ISSUE TITLE
Nested grids are not reachable with keyboard 

### ISSUE THREAD

**Issue by arcady-zherdev** [bryntum/support#6950]
Can be reproduced on `master-detail` in the Grid. Pressing `[TAB]` or `[SHIFT-TAB]` traverses root level grid cells only.

PS Seems you can navigate with arrow keys down to a nested grid but then you cannot navigate back to upper levels.
